### PR TITLE
Resolve Add Command UI-related Bugs

### LINKED ISSUE #116: Bug: Missing fields do not return a error message in the UI
Steps to reproduce:
1. Start application
2. Enter this into command line `add /com Google /desc Design user-friendly interfaces in Figma /status pending /poc john doe /email johndoe@hotmail.com /phone 12345678 /loc remote`, which is an invalid add command, missing the required `role` field entirely
3. Expected error output message, but no output error message is shown in the UI

### PR DESCRIPTION
**Pull request by HM33-Stu**
Resolves #116 
v1.2b Issue

### CODE REVIEW

**Comment by HM33-Stu** [comment]
Resolves the following raised issues

**Issue 1: Missing Fields Do Not Return Error Message** (Raised by @jieqiboh)
Now, not typing in the full internship entry with all fields will return the standard error message as below.
Tried it out using exact command as shared by @jieqiboh in Issue # 116
image





**Issue 2: Sample Command Is Invalid** (Raised by @LWS49)
Now, the sample command displayed, when entered as an input, creates a new internship entry/record
image

image

File: src/main/java/seedu/address/logic/commands/InternshipAddCommand.java [hunk 1]
```diff
@@ -25,22 +25,23 @@ public class InternshipAddCommand extends InternshipCommand {
...
+            + PREFIX_DESCRIPTION + " create new recommendation engine "
+            + PREFIX_STATUS + " ongoing "
+            + PREFIX_CONTACT_NAME + " jane yeo "
+            + PREFIX_CONTACT_EMAIL + " hr@tiktok.com "
+            + PREFIX_CONTACT_NUMBER + " 9089030 "
+            + PREFIX_LOCATION + " remote "
+            + PREFIX_ROLE + " Software Intern";
 
     public static final String MESSAGE_SUCCESS = "New internship added: %1$s";
     public static final String MESSAGE_DUPLICATE_INTERNSHIP = "This internship already exists in the address book";
```
Comment (CaptainKeqing) [inline src/main/java/seedu/address/logic/commands/InternshipAddCommand.java:47]: Typo, can change to Internship List

**Review by CaptainKeqing — COMMENTED** [review]
lgtm, 1 small nit typo

**Review by CaptainKeqing — APPROVED** [review]
LGTM, I fixed the small typo

**Pull request merged**